### PR TITLE
scp rebranding text changes

### PR DESCRIPTION
**Pull request by Juliana-Morais**

### CODE REVIEW

File: tutorials/cp-aibus-ber-service-instance/cp-aibus-ber-service-instance.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ primary_tag: topic>machine-learning
 
 [ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
+1. In your web browser, open the [SAP BTP trial cockpit](https://cockpit.hanatrial.ondemand.com/).
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-ber-service-instance/cp-aibus-ber-service-instance.md:23]: Trial

File: tutorials/cp-aibus-dar-booster-postman/cp-aibus-dar-booster-postman.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ primary_tag: topic>machine-learning
 
 [ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
+1. In your web browser, open the [SAP BTP trial cockpit](https://cockpit.hanatrial.ondemand.com/).
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-dar-booster-postman/cp-aibus-dar-booster-postman.md:23]: Trial

File: tutorials/cp-aibus-dar-service-instance-booster/cp-aibus-dar-service-instance-booster.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ primary_tag: topic>machine-learning
 
 [ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
+1. In your web browser, open the [SAP BTP trial cockpit](https://cockpit.hanatrial.ondemand.com/).
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-dar-service-instance-booster/cp-aibus-dar-service-instance-booster.md:23]: Trial

File: tutorials/cp-aibus-dar-service-instance/cp-aibus-dar-service-instance.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ primary_tag: topic>machine-learning
 
 [ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
+1. In your web browser, open the [SAP BTP trial cockpit](https://cockpit.hanatrial.ondemand.com/).
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-dar-service-instance/cp-aibus-dar-service-instance.md:23]: Trial

File: tutorials/cp-aibus-dc-service-instance/cp-aibus-dc-service-instance.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ primary_tag: topic>machine-learning
 
 [ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
+1. In your web browser, open the [SAP BTP trial cockpit](https://cockpit.hanatrial.ondemand.com/).
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-dc-service-instance/cp-aibus-dc-service-instance.md:23]: Trial

File: tutorials/cp-aibus-dc-swagger-ui/cp-aibus-dc-swagger-ui.md [hunk 6]
```diff
@@ -15,7 +15,7 @@ primary_tag: topic>machine-learning
...
+In the trial version of Document Classification, however, a predefined and pre-trained machine learning model is made available for all users. Thus, the classification categories cannot be customized. This tutorial gives you an introduction to the service. If you want to try out the Document Classification training endpoints to create your own classification model, you can use an Enterprise Account (see [Trial Accounts](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/046f127f2a614438b616ccfc575fdb16.html) versus [Enterprise Accounts](https://help.sap.com/viewer/3504ec5ef16548778610c7e89cc0eac3/Cloud/en-US/171511cc425c4e079d0684936486eee6.html)) or an enterprise global account on SAP BTP.
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-dc-swagger-ui/cp-aibus-dc-swagger-ui.md:18]: Please mention the full product name at least once at the beginning of a tutorial before using the abbreviation.

File: tutorials/cp-aibus-dox-booster-app/cp-aibus-dox-booster-app.md [hunk 7]
```diff
@@ -20,7 +20,7 @@ primary_tag: topic>machine-learning
 
 [ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
+1. In your web browser, open the [SAP BTP trial cockpit](https://cockpit.hanatrial.ondemand.com/).
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-dox-booster-app/cp-aibus-dox-booster-app.md:23]: Trial

File: tutorials/cp-aibus-dox-booster-key/cp-aibus-dox-booster-key.md [hunk 8]
```diff
@@ -20,7 +20,7 @@ primary_tag: topic>machine-learning
 
 [ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
+1. In your web browser, open the [SAP BTP trial cockpit](https://cockpit.hanatrial.ondemand.com/).
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-dox-booster-key/cp-aibus-dox-booster-key.md:23]: Trial

File: tutorials/cp-aibus-dox-data-api-bus/cp-aibus-dox-data-api-bus.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Use Machine Learning to Enrich Business Data with Swagger UI
-description: Create, update, list and delete enrichment data using the Data API business entity from Document Information Extraction, one of the SAP AI Business Services in SAP Cloud Platform.
+description: Create, update, list and delete enrichment data using the Data API business entity from Document Information Extraction, one of the SAP AI Business Services in SAP BTP.
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-dox-data-api-bus/cp-aibus-dox-data-api-bus.md:3]: Please use the full product name at least once before you use the abbreviation.

File: tutorials/cp-aibus-dox-data-api-emp/cp-aibus-dox-data-api-emp.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Use Machine Learning to Enrich Employee Data with Swagger UI
-description: Create, update, list and delete enrichment data using the Data API employee entity from Document Information Extraction, one of the SAP AI Business Services in SAP Cloud Platform.
+description: Create, update, list and delete enrichment data using the Data API employee entity from Document Information Extraction, one of the SAP AI Business Services in SAP BTP.
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-dox-data-api-emp/cp-aibus-dox-data-api-emp.md:3]: Please use the full product name at least once before you use the abbreviation.

File: tutorials/cp-aibus-dox-service-instance-booster/cp-aibus-dox-service-instance-booster.md [hunk 11]
```diff
@@ -20,7 +20,7 @@ primary_tag: topic>machine-learning
 
 [ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
+1. In your web browser, open the [SAP BTP trial cockpit](https://cockpit.hanatrial.ondemand.com/).
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-dox-service-instance-booster/cp-aibus-dox-service-instance-booster.md:23]: Trial

File: tutorials/cp-aibus-dox-service-instance/cp-aibus-dox-service-instance.md [hunk 12]
```diff
@@ -20,7 +20,7 @@ primary_tag: topic>machine-learning
 
 [ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
+1. In your web browser, open the [SAP BTP trial cockpit](https://cockpit.hanatrial.ondemand.com/).
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-dox-service-instance/cp-aibus-dox-service-instance.md:23]: Trial

File: tutorials/cp-aibus-dox-ui-sub/cp-aibus-dox-ui-sub.md [hunk 13]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Subscribe to Document Information Extraction Trial UI
-description: Get access to the Document Information Extraction user interface application, using the SAP Cloud Platform trial cockpit.
+description: Get access to the Document Information Extraction user interface application, using SAP BTP Trial.
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-dox-ui-sub/cp-aibus-dox-ui-sub.md:3]: Please use the full product name at least once before you use the abbreviation.

File: tutorials/cp-aibus-ior-service-instance/cp-aibus-ior-service-instance.md [hunk 14]
```diff
@@ -20,7 +20,7 @@ primary_tag: topic>machine-learning
 
 [ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
+1. In your web browser, open the [SAP BTP trial cockpit](https://cockpit.hanatrial.ondemand.com/).
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-ior-service-instance/cp-aibus-ior-service-instance.md:23]: Trial

File: tutorials/cp-aibus-sti-booster-key/cp-aibus-sti-booster-key.md [hunk 15]
```diff
@@ -20,7 +20,7 @@ primary_tag: topic>machine-learning
 
 [ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
+1. In your web browser, open the [SAP BTP trial cockpit](https://cockpit.hanatrial.ondemand.com/).
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-sti-booster-key/cp-aibus-sti-booster-key.md:23]: Trial

File: tutorials/cp-aibus-sti-service-instance/cp-aibus-sti-service-instance.md [hunk 16]
```diff
@@ -20,7 +20,7 @@ primary_tag: topic>machine-learning
 
 [ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
+1. In your web browser, open the [SAP BTP trial cockpit](https://cockpit.hanatrial.ondemand.com/).
```
Comment (MichaelCzcz) [inline tutorials/cp-aibus-sti-service-instance/cp-aibus-sti-service-instance.md:23]: Trial

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
There are many issues with branding. Most of them seem to be C&P typos.

Please check the [SAP BTP Rebranding Guidelines](https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?spaceKey=NDW&title=SAP+BTP%3A+Rebranding+Guidelines) for more information.

**Comment by Juliana-Morais** [comment]
Fixed the rebranding issues.

**Pull request merged**